### ISSUE TITLE
add detailed info for each item

### ISSUE THREAD

**Issue by vabruzzo** [vga-/wave-auto-a11y#9]
check waves documentation API

**Comment by vabruzzo** [comment]
added. however, a11y doesn't work with popover ids...look into this, possibly open issue on react-bootstrap

**Comment by vabruzzo** [comment]
switched to modals. popovers ina11y
